### PR TITLE
Batch raw secrets api

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Description 📣

1. API endpoints for batch raw secrets
2. Docs for the same

## Type ✨

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Documentation

# Tests 🛠️

```sh
# Here's some code block to paste some code snippets
```

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/getting-started/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/getting-started/code-of-conduct). 📝

### CODE REVIEW

File: backend/src/server/routes/v3/secret-router.ts [hunk 1]
```diff
@@ -1656,4 +1656,263 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
...
+    },
+    schema: {
+      description: "Create many secrets",
+      security: [
+        {
+          bearerAuth: []
+        }
+      ],
+      body: z.object({
+        workspaceId: z.string().trim().describe(RAW_SECRETS.CREATE.workspaceId),
```
Comment (maidul98) [inline backend/src/server/routes/v3/secret-router.ts:1674]: project slug?

File: backend/src/server/routes/v3/secret-router.ts [hunk 2]
```diff
@@ -1656,4 +1656,263 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
...
+          .array()
+          .min(1)
+      }),
+      response: {
+        200: z.object({
+          secrets: secretRawSchema.array()
+        })
+      }
+    },
+    onRequest: verifyAuth([AuthMode.JWT, AuthMode.API_KEY, AuthMode.SERVICE_TOKEN, AuthMode.IDENTITY_ACCESS_TOKEN]),
```
Comment (maidul98) [inline backend/src/server/routes/v3/secret-router.ts:1872]: let's remove API and service tokens and only have JWT and identity for these new endpoints

File: backend/src/server/routes/v3/secret-router.ts [hunk 3]
```diff
@@ -1656,4 +1656,263 @@ export const registerSecretRouter = async (server: FastifyZodProvider) => {
...
+        environment: z.string().trim().describe(RAW_SECRETS.DELETE.environment),
+        secretPath: z
+          .string()
+          .trim()
+          .default("/")
+          .transform(removeTrailingSlash)
+          .describe(RAW_SECRETS.DELETE.secretPath),
+        secrets: z
+          .object({
+            secretKey: z.string().trim().describe(RAW_SECRETS.DELETE.secretName)
```
[the remark this answers is not in the document]
Comment (akhilmhdh) [reply backend/src/server/routes/v3/secret-router.ts:1861]: The response is keeping it similar to other raw operations. Its better to not change the response as same version raw operations returning different keys  could affect usability

[the remark this answers is not in the document]
Comment (maidul98) [reply backend/src/server/routes/v3/secret-router.ts:1861]: okay, fair enough

File: backend/src/server/routes/v3/secret-router.ts (hunk 2 above)
Comment (maidul98) [reply backend/src/server/routes/v3/secret-router.ts:1872]: okay, let's keep it same until full non e2ee initiative

**Review by maidul98 — APPROVED** [review]
lgtm, please test it out before informing customer about this release

**Pull request merged**